ksql-engine/src/main/java/io/confluent/ksql/planner/plan/JoinNode.java

```diff
@@ -74,8 +74,8 @@ public JoinNode(
       @JsonProperty("type") final JoinType joinType,
       @JsonProperty("left") final PlanNode left,
       @JsonProperty("right") final PlanNode right,
-      @JsonProperty("leftKeyFieldName") final String leftKeyFieldName,
-      @JsonProperty("rightKeyFieldName") final String rightKeyFieldName,
+      @JsonProperty("leftJoinFieldName") final String leftJoinFieldName,
```

Comment:
I don't think we should change the names here - wouldn't that be backwards incompatible? (same for getters/setters)

Reply:
This is never serialized to JSON. We should drop the Json annotations.